tests/commands/dev/conftest.py

```diff
@@ -18,22 +19,35 @@ def __init__(self, *args, **kwargs):
         super().__init__(*args, apps=[], **kwargs)
 
         self.actions = []
+        self.env = dict(a=1, b=2, c=3)
 
     def install_dev_dependencies(self, app, **kwargs):
         self.actions.append(('dev_dependencies', app.app_name, kwargs))
 
-    def run_dev_app(self, app, **kwargs):
-        self.actions.append(('run_dev', app.app_name, kwargs))
+    def get_environment(self, app):
+        return self.env
+
+    def run_dev_app(self, app, env, **kwargs):
+        self.actions.append(('run_dev', app.app_name, kwargs, env))
+        self.env.update(env)
         return full_kwargs({
-            'run_dev_state': app.app_name
+            'run_dev_state': app.app_name,
+            'env': env
         }, kwargs)
 
 
 @pytest.fixture
-def dev_command(tmp_path):
+def dummy_dev_command(tmp_path):
```

Comment:
This rename causes a lot of code churn, but doesn't significantly increase readability. Since the 'replacement' `dev_command` is only used in `test_run_dev_app`, I'd suggest moving the "new" fixture into that specific test.